src/term/parser/parser.rs

```diff
@@ -130,14 +130,24 @@ where
       .map(|(((fst, snd), val), next)| Term::Dup { fst, snd, val: Box::new(val), nxt: Box::new(next) })
       .boxed();
 
-    // let x = body; next
+    // (x, y)
+    let pair = term
+      .clone()
+      .then_ignore(just(Token::Comma))
+      .then(term.clone())
+      .delimited_by(just(Token::LParen), just(Token::RParen))
+      .map(|(fst, snd)| Term::Tup { fst: Box::new(fst), snd: Box::new(snd) })
+      .boxed();
+
+    // let a = ...
+    // let (a, b) = ...
     let let_ = just(Token::Let)
-      .ignore_then(name())
+      .ignore_then(pat())
```

Comment:
Please call this `let_pat` for consistency and avoid clashing with rule patterns. (At some point I think we'll merge both)